.github/workflows/ci.yml

```diff
@@ -210,6 +214,7 @@ jobs:
           poetry install --all-extras
           poetry run pip install protobuf==3.19.0
           poetry run pip install tensorflow==2.11.0
+          sudo apt-get install --no-install-recommends ffmpeg
```

Comment:
`--no-install-recommends` what is this doing ?

Reply:
Don't let apt to install some "recommended" (not needed) packages, to save a little bit of CI time.